kasa/smartprotocol.py

```diff
@@ -157,15 +147,50 @@ async def _execute_query(self, request: Union[str, Dict], retry_count: int) -> D
             self._host,
             _LOGGER.isEnabledFor(logging.DEBUG) and pf(smart_request),
         )
-        response_data = await self._transport.send(smart_request)
+        resp_dict = await self._transport.send(smart_request)
```

Comment:
Should we rather keep the original name?

Reply:
Reverted